drivers/storage/scaleio/tests/test-env-up.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This script launches infrastructure required for testing the ScaleIO storage driver.
+# It spins up VPC and EC2 instance so AWS account owner will get charged for time
+# that resources will be running.
+
+set -e
+# set -x
+
+AWS_ACCESS_KEY="$1"
+AWS_SECRET_KEY="$2"
+CF_STACK_NAME="$3"
+LAUNCH_KEY_NAME="$4"
+
+# Make sure that aws cli is installed
+hash aws 2>/dev/null || {
```

Comment:
Hi @dvonthenen,

Ah, I never had them update those. I apologize for the confusion. I wouldn't mind making it part of Travis if there was a way to provide you the ability to do it.

Hmmm, let me think about it. We could make that part of the custom Travis config so that the dependency section includes test dependencies?